### ISSUE TITLE
Accessibility fail

### ISSUE THREAD

**Issue by chasm** [oven-sh/bun#927]
### Version

0.1.5

### Platform

Darwin MacBook-Pro-2.local 21.6.0 Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:25 PDT 2022; root:xnu-8020.140.41~1/RELEASE_X86_64 x86_64

### What steps will reproduce the bug?

`bun create react`

### How often does it reproduce? Is there a required condition?

Every time.

### What is the expected behavior?

Should generate output that meets accessibility guidelines.

### What do you see instead?

Two errors:

- Document should have one main landmark
- Page should contain a level-one heading

### Additional information

This just makes me sad. It is 2022! Are we really still using templates that create output with obvious and easily-fixed accessibility issues? Do we really just not give a shit about accessibility at all? Doesn't anyone check this?

The template already uses CSS for styling, so why would you choose an h3 heading instead of h1? The only reason could be to set font size, **which should be set with CSS**. Sorry, but this was well known two decades ago.

And then to use a bland div element without a role instead of a true landmark. Come on!

This is easily fixed. Change the h3 element to h1 and set the size you want in CSS. You set font-size on the header with `font-size: calc(10px + 2vmin);`. Surely you can get this right.

Then either change `<div className="App">` to `<main className="App">` or give it a role: `<div className="App" role="main">` (in which instance the header element needs to be changed, but then it is not really a "header" is it?).

Done. I haven't tried the other templates but I suspect they have the same problems. If the templates **start** with crap HTML, how many devs will bother to fix the HTML? How many will got on to build crap, inaccessible web apps?

Please. Get Bun off on the right footing. [Learn more](https://www.w3.org/WAI/ARIA/apg/patterns/).

**Comment by chasm** [comment]
Also, I strongly recommend a pro version of [axe DevTools](https://www.deque.com/axe/devtools/) as the best way to check accessibility and get answers for how to fix issues. And no, I do not have any connection with Dequeue. Lighthouse works, too, but is not as thorough on accessibility.

**Comment by Jarred-Sumner** [comment]
I wish I knew more about accessibility.

You're welcome to submit a PR fixing the issues

**Comment by chasm** [comment]
Deal! Awesome.

**Comment by chasm** [comment]
I created a pull request. Very simple changes.

I suspect you're as busy as I am, so don't bother to study accessibility. Just download the free [Dequeue axe DevTools](https://www.deque.com/axe/devtools/) browser extension (there's one for Edge as well) and run the linter on any HTML output you generate.

It will list all the accessibility issues and explain how to fix them. Usually, it's quite simple. Good luck.

You'll end up picking up some accessibility knowledge as you go.

**Comment by xhyrom** [comment]
Closed by #932
